CommunityFridgeMapApi/tests/unit/test_db_item.py

```diff
@@ -0,0 +1,14 @@
+from CommunityFridgeMapApi.dependencies.python.db import DB_Item
+
+import unittest
+
+
+class DB_Item_Test(unittest.TestCase):
+    def test_remove_whitespace(self):
+        result = DB_Item.remove_whitespace("   h  i th er  h o w    ")
```

Comment:
Isn't `remove_whitespace` slightly misleading? Because whitespaces are still there in the string?

Reply:
yeah maybe it should be "remove_extra_whitespace"

Reply:
done: 4909a4d969cd39b65cf003f898c624305c210b76